_sass/meet-the-program-coordinator.css

```diff
@@ -0,0 +1,64 @@
+* {
+  box-sizing: border-box;
+	font-family: 'Open Sans', sans-serif;
+}
+
+.box-content {
+  margin: 0;
+  padding: 0;
+  background-color: #fff;
+  color: #272b34;
+}
+.box-content {
+  margin-top: 250px;
+  display: block;
+  margin-bottom: 250px;
+  background-color: inherit;
+}
+.br-left > h1 {
+  text-align: center;
+	text-transform: uppercase;
+}
+
+button {
+  background-color: #9e1f63;
+  border-radius: .5rem;
+  border-width: 0;
+  color: #fff;
+  cursor: pointer;
+  font-size: 1.1rem;
+  padding: .5rem 1.5rem;
+  text-decoration: none;
+}
+
+.youth-managed-farm__button:not(:last-of-type) {
+  margin-right: 12px; 
+}
+.br-right {
+	margin-top: 25px;
+}
+img {
```

Comment:
This rule is too general. We'll need to use a classname on that image and reference here.